libs/native-types/src/type_parameter.rs

```diff
@@ -0,0 +1,55 @@
+use serde::{Deserialize, Serialize};
+use std::{fmt, io, str::FromStr};
+
+/// A parameter given to the SQL Server type. In many cases a number, but for
+/// some variants could also be `max`, allowing the value to be taken from the
+/// row to a larger heap.
+#[derive(Debug, Clone, PartialEq, Copy, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum TypeParameter {
```

Comment:
The name `TypeParameter` sounds very generic but it is specific to MSSQL. Can we rename it to `MssqlTypeParameter` and put it into the module for MSSQL?

Reply:
we can also refer into it with the module `mssql::TypeParameter`, which looks much cleaner, and we don't need to repeat `MsSqlTypeParameter` inside the mssql module.